RELEASE.md

```diff
@@ -0,0 +1,253 @@
+
+## Who is responsible for the release?
+
+The release coordinator is elected at the [community meeting] at the beginning of the release cycle.
+The release coordinator may be any volunteer from the community - a call for volunteers will be issued one week before the next community meeting via the community Slack and mailing list.
+
+### Release Coordinator Responsibilities
+
+- The release coordinator is the person who is ultimately responsible for ensuring that the release is completed on time, and with the desired payload.
+- The release coordinator follows the release process outlined in [#what-is-the-release-process].
+- The release coordinator ensures that all community members are following the correct process.
+- While not a maintainer, the release coordinator is an important role in the community and should be afforded the respect and trust of the maintainers.
+- The release coordinator is empowered to make decisions about the release that are in the best interest of the community.
+  - For example, the release coordinator may choose to delay the release if the payload is not ready or if the quality of the release is not high enough.
+- The release coordinator is responsible for ensuring that progress is being made on the release and regularly updates the community on the progress of the release during the [community meeting].
+- If decisions are required that are outside the scope of the release coordinator's responsibilities, the release coordinator will seek the advice of the maintainers.
+- If the release coordinator is unable to fulfill their responsibilities, they may step down and a new release coordinator will be elected.
+
+## What goes into a release?
+
+A release is a collection of changes that are considered stable and ready for production use.
+These changes can be broken down into the following categories:
+
+- **New features** - new features require an approved OKEP.
+- **Minor changes** - small changes to improve the project.
+- **Bug fixes** - fixes for bugs.
+- **Documentation** - improvements to the documentation.
+- **Testing** - improvements to the testing or CI workflows.
+
+At the [community meeting], we will discuss all approved OKEPs and issues that are targeted for the release and select a maximum of **5** OKEPs for the release.
+This is limited to ensure that:
+
+- There is sufficient development time.
+- Developer focus is not spread too thin.
+- Community feedback is solicited and considered during the release cycle.
+- The release is not too large with too many potential breaking changes.
+
+> [!TIP]
+> As a future enhancement to the process we may wish to consider using T-shirt sizes to the OKEPs instead (XS<->XL). Each size can be represented as a number: XS=1, S=2, M=3, L=4, XL=5. Given release capacity of 10, we could select OKEPs whose combined size is less than or equal to 10.
+
+## How long are releases supported?
+
+OVN-Kubernetes is supported until the next major or minor release.
```

Comment:
wait this mean 0 maintainability for products using this. 

... not 0, but only 3 months of maintained code. 

That's too little imho. FWIW, kubernetes does support N-3 releases.